src/menu/cart_load.c

```diff
@@ -26,7 +23,7 @@
  * 
  * @return true if the 64DD is connected, false otherwise.
  */
-static bool is_64dd_connected (void) {
+bool is_64dd_connected (void) {
```

Comment:
I have been thinking about this change, and think it is worth while to to ask if `physical dd hardware detection` can be added to the libdragon SDK. It would be advantagous over this change. Please feel free to suggest it in the N64brew discord.